api/core/tools/provider/builtin/dingtalk/tools/dingtalk_group_bot.yaml

```diff
@@ -0,0 +1,52 @@
+identity:
+  name: dingtalk_group_bot
+  author: Bowen Liang
+  label:
+    en_US: Send Group Message
+    zh_Hans: 发送群消息
+    pt_BR: Send Group Message
+  icon: icon.svg
+description:
+  human:
+    en_US: Sending a group message on DingTalk via the webhook of group bot
+    zh_Hans: 通过钉钉的群机器人webhook发送群消息
+    pt_BR: Sending a group message on DingTalk via the webhook of group bot
+  llm: A tool for sending messages to a chat group on DingTalk(钉钉) .
+parameters:
+  - name: access_token
+    type: string
```

Comment:
Please use this type `type: secret-input`.

Reply:
Thanks. Resolved.